nebula3/data/ResultSet.py

```diff
@@ -192,6 +192,187 @@ def rows(self):
             return []
         return self._data_set_wrapper.get_rows()
 
+    def dict_for_vis(self):
```

Comment:
what about the result value is just a path?

Reply:
Yes, it hands Path use cases, too(even a list of paths ideally), it literally hands all results that are feasible to be rendered.

Reply:
For Paths, it'll extract edge list and node list, too. For all graph Vis Lib, edge list and node list are all they needed.